resources/views/welcome.blade.php

```diff
@@ -1,11 +1,7 @@
 <x-layout>
-    <p>Coming soon, to a screen near you...</p>
------------------------
     @auth
-        <p>Hello, {{ Auth::user()->username }}! <a href="{{ route('logout') }}">Logout</a></p>
-
-        Check out our <a href="{{ route('token') }}">shiny new API tokens</a>!
+     DONT know how to add the redirect here!
```

Comment:
This is not a redirect. I think the page should just display different content for people who are logged in, and this content should be here.

Reply:
See https://github.com/wmde/wikidata-mismatch-finder/commit/5247f4f74645311ecfb758b556c0de2b5e16eb3e for an attempt on how it can be done.

Reply:
Thanks @Silvan-WMDE . I have merged the commit into this one.